packages/tocco-ui/src/SearchBox/SearchBox.js

```diff
@@ -2,9 +2,8 @@ import PropTypes from 'prop-types'
 import React from 'react'
```

Comment:
SearchBox styling will also affect the searchbox used in Detail > Relation Preview. But there we have a litte imperfection:
![image](https://user-images.githubusercontent.com/7300481/70227292-9f901b00-1752-11ea-9c07-70b21dc7f50a.png)